Makefile

```diff
@@ -170,7 +170,7 @@ dns-smoke-test:
 # build docker images for multiple architectures
 # useful for CI
 .PHONY: docker-build-multi
-docker-build-multi: test
+docker-build-multi:
```

Comment:
locally it is properly guarding the docker build by running the tests.

Why the tree is getting dirty in the first place?

Reply:
generated crd.

Reply:
so this generation-based deviation is a problem on multiple levels.

Still not a reason to disable the tests

Reply:
docker-build-multi is used in CI only as it makes no sense to build arm64 locally as we discussed while implementing amr64 support. CI runs tests in other jobs, if not a generator it would simply just extra time. Eitherway, will implement docker build by using docker-buildx action. Hold on ;)